pkg/southbound/synchronizer/factory.go

```diff
@@ -57,13 +58,19 @@ func Factory(topoChannel <-chan *topodevice.ListResponse, opStateChan chan<- eve
 			} else {
 				mStateGetMode = modelregistry.GetStateMode(mPlugin.GetStateMode())
 			}
+			mReadWritePaths, ok := modelRegistry.ModelReadWritePaths[modelName]
+			if !ok {
+				log.Warnf("Cannot check for read write paths for target %s with %s because "+
```

Comment:
Following this code as best as I can to the end, it looks to me like jsonpaths.CorrectJSONPaths will get a nil dereference if mReadWritePaths is nil. I think you have to set it to an empty map here.